assets/css/styles.css

```
@@ -1,46 +1,180 @@
-@import url('https://fonts.googleapis.com/css?family=Rubik:300,500&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Barlow:300,500,700&display=swap');
 
 * {
-  font-family: 'Rubik', sans-serif !important;
-  font-weight: 300 !important;
+  font-family: 'Barlow', sans-serif !important;
 }
-
-.title {
-  font-size: 2.5rem;
-  font-weight: 500 !important;
-  margin-top: 1rem;
-  margin-bottom: 0.1rem;
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+.logo {
+  width: 7rem;
+  height: 4rem;
+}
+.nav-text {
+  color: #22213d;
+}
+.holding {
+  background-image: url('../img/holding.png');
+  background-size: contain;
+  height: 28rem;
+  border-radius: 0.5rem;
+  width: 100%;
+  padding-left: 7rem;
+}
+.drop-text {
+  font-weight: 600;
 }
 
-.form-container {
-  margin: auto;
-  margin-top: 2.6rem;
+.coming-soon {
+ font-weight: 800;
+ font-size: 3rem;
+ letter-spacing: 0.1rem;
+ color: #ffffff;
+ text-shadow: 4px 4px #161d3f;
+
 }
 
-.logo {
-  margin: auto;
-  height: 3.5rem;
-  width: 6rem;
+.ball-full-text {
+  color: #161d3f;
+  font-size: 1.8rem;
+  font-weight: bold;
+  padding-top: 3rem;
+  padding-right: 3rem;
+  letter-spacing: 0.05rem;
 }
 
 .form-control {
+  border: none;
+  border-bottom: 2px solid #5775fa;
   height: 3rem;
-  font-size: 0.8rem;
   border-radius: 0;
-  color: black;
+  color: #c0ccda;
+  background-color: #161d3f;
 }
 
-.submit-btn {
-  border-radius: 0;
-  background: #990d0d;
+.form-control:focus {
+  outline: none;
+  color: #c0ccda;
+  background-color: #161d3f;
+  border-bottom: solid #5775fa;
+}
+
+.contact-form {
+  background: #161d3f;
+  border-radius: 0.5rem;
   color: #ffffff;
-  text-transform: uppercase;
-  font-size: 0.85rem;
+  top: -3.2rem;
+  padding: 1rem;
```

Can we increase the top padding please?
```suggestion
  padding: 2rem 1rem 1rem;
```